acceptance_tests/features/pages/reporting_units.py

```diff
@@ -0,0 +1,16 @@
+from acceptance_tests import browser
```

Comment:
I think this file should be called `reporting_units_search` if we're going to have a file in this folder per page

Reply:
It's the /reporting-units page which is why I named it reporting_units.py the reporting unit details page should be called reporting_units_details.py to match the current pattern we have with collection_exercise.py and collection_exercise_details.py.